resources/views/home.twig

```diff
@@ -1,7 +1,7 @@
 {% extends "layouts/default.twig" %}
 
 {% block header %}
-    <header id="hero" class="bg-cover p-24" style="background:linear-gradient(rgba(0, 0, 0, 0.6), rgba(0, 0, 0, 0.6)), url(/assets/img/venue.jpg) no-repeat center -300px fixed;">
+    <header id="hero" class="opencfp-hero">
```

Comment:
Extracting a class `opencfp-hero` here so we don't need to apply multiple classes.